packages/sdk/src/lib/dataProtectorCore/processProtectedData.ts

```diff
@@ -100,20 +141,72 @@ export const processProtectedData = async ({
       workerpool: underMaxPriceOrders.workerpoolorder.workerpool,
       params: {
         iexec_input_files: vInputFiles,
-        iexec_developer_logger: true,
         iexec_secrets: secretsId,
         iexec_args: vArgs,
       },
     });
 
     const requestorder = await iexec.order.signRequestorder(requestorderToSign);
 
-    const { dealid } = await iexec.order.matchOrders({
+    const { dealid, txHash } = await iexec.order.matchOrders({
       requestorder,
       ...underMaxPriceOrders,
     });
+    vOnStatusUpdate({
+      title: 'PROCESS_PROTECTED_DATA_REQUESTED',
```

Comment:
I'd like to keep a status with both `{ isDone: false }` then same status with `{ isDone: true }`

There is  no `{ isDone: false }` for this one.

Also I'd renamed it to: REQUEST_TO_PROCESS_PROTECTED_DATA
and that this event wraps 3 async calls:
- iexec.order.createRequestorder()
- iexec.order.signRequestorder()
- iexec.order.matchOrders()

wdyt?

Reply:
commit: d06b52ac02242b9248b18303dc4ce9665dac4fe3